extensions/amp-fit-text/0.1/build-dom.js

```diff
@@ -33,6 +35,26 @@ export function buildDom(element) {
   return {content, contentWrapper, measurer};
 }
 
+/**
+ * Returns all of the needed ivars from a server rendered element.
+ * @param {!Element} element
+ * @return {{content: !Element, contentWrapper: !Element, measurer: !Element}}
+ */
+export function queryDom(element) {
+  const content = element.querySelector(
+    `.${escapeCssSelectorIdent(CONTENT_CLASS)}`
+  );
+  const contentWrapper = element.querySelector(
+    `.${escapeCssSelectorIdent(CONTENT_WRAPPER_CLASS)}`
+  );
+  const measurer = element.querySelector(
+    `.${escapeCssSelectorIdent(MEASURER_CLASS)}`
+  );
```

Comment:
Would it make sense to define a `elementsByClass(element, className)` helper in `#core/dom/query`?